### ISSUE TITLE
[BUG] 0.3.2 on Izzy not starting, black screen

### ISSUE THREAD

**Issue by Der-Flip** [Myzel394/locus#28]
## OS

* [x] Android
* [ ] iOS

## Android specific
Redmi9C
Android 10 QP1A.190711.020
MIUI 12.0.4 Global Stable

## Describe the bug
App loaded from Izzy doesn't start. Black screen only.

## To Reproduce
1. Install app from Izzy
2. Open App
3. ...

## Expected behavior
App opens

## Additional context
Hope this works

![2023-05-29-21-23-48_myzel.txt](https://github.com/Myzel394/locus/files/11593949/2023-05-29-21-23-48_myzel.txt)

```
05-29 21:17:52.775 W/BroadcastQueue(1225): Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_REMOVED dat=package:app.myzel394.locus flg=0x4000010 (has extras) } to com.google.android.apps.maps/com.google.android.libraries.social.peoplekit.thirdparty.viewcontrollers.ThirdPartyReceiver
```

**Comment by Myzel394** [comment]
@Der-Flip could you try installing the GMS version? https://github.com/Myzel394/locus/releases/tag/v0.3.2

**Comment by Myzel394** [comment]
ohhh no I think I know what's the error, a library (`permission_handler`) depends on gms afaik, I'll push a new hotfix out so that you can use the app, but I think you will need to use the GMS version at the moment.

**Comment by Der-Flip** [comment]
So the armebi makes the phone behave weirdly with the same main symptom.

The universal same symptom, only error:

```
05-29 22:39:57.528 E/flutter (30071): #4      main (package:locus/main.dart:28)
```

**Comment by Myzel394** [comment]
@Der-Flip I have pushed the new release. Files should be ready in ~20 minutes. Could you check if either the FLOSS or the GMS version work for you?

**Comment by Myzel394** [comment]
Okay both versions work for me on my Xiaomi device. @IzzySoft you may need to add the non-free dependency anti feature back :/ can you check if that's the case?

**Comment by Der-Flip** [comment]
Both GM's and floss start just fine.

GMS wishes access to Google accuracy, denying it no fix is attempted to retrieve.

Floss works tries to retrieve fix. (My home is horrible with GPS though, so if I don't get back to you regarding this on the other issue tomorrow, just assume it works :'D)

**Comment by Myzel394** [comment]
@Der-Flip sounds good, feel free to open another issue if the GPS does not work. I'm very sorry that the app does not work flawlessly for you. It's still in beta, so these kind of bugs shouldn't discourage you from it. I hope you understand this :C

**Comment by Der-Flip** [comment]
Oh I'm not discouraged. I may not use it as my primary driver but I am kinda here for the journey, the experimenting and helping you to get the app out there by improving it (from an idiot's experience kinda perspective) because I think the app really has potential.

I just hope I'm not getting on your nerves with my frequent and many (and very clueless) messages 😳

**Comment by Myzel394** [comment]
> I just hope I'm not getting on your nerves with my frequent and many (and very clueless) messages flushed

No actually quite the opposite - I'm happy that you informed me about this issue (and also the other ones) as I would not have realized it myself! I'm using the dev version on my devices, so I don't use the F-Droid versions :/ And I also never heard of this log thing before you mentioned it and attached to log to it. btw, how did you do that? I do not really have any logs in Locus yet :sweat_smile:  but after your comment regarding the logs, I will definitely need to add some to make debugging the app easier.

**Comment by Der-Flip** [comment]
I use [matlog](https://github.com/pluxoid/matlog) to get the logcats (catlogs...never know which way round).

Not very confident in what I'm doing but calming my conscience that at least I try to help the Devs attempting to get the logs :'D

**Comment by IzzySoft** [comment]
> you may need to add the non-free dependency anti feature back

My framework should alert me about that (if it detects a non-free library). If the app works fine without GMS, and there's no such lib included, it won't be needed. Until now, I didn't receive an alert from my checker.

> logcats (catlogs...never know which way round).

You log the cats, you can't cat the logs (not with a :cat: at least :see_no_evil: – and the `cat` command would glue things together – aww, mnemonic aid isn't what it used to be, and I better don't try to bring catalogs into this game…).

Well, it's logcat (at the command line: `adb logcat`).

**Comment by Der-Flip** [comment]
I appreciate that little Eselsbrücke (too lazy to Google an English translation rn 😳) and I do appreciate the effort, just don't be mad if in two hours I call it catlogs again 😂

**Comment by IzzySoft** [comment]
Eh, that's the Internet, right? So what would that be without Katzenbilder? Half empty I guess – I've heard 95% of the traffic is either spam, tracking or cat pics :speak_no_evil: :dash: 

TL;DR: Call your cat whatever you want. It's either the black cat, the orange cat – or the log cat, ha! There you have your Eselsbrücke! :rofl:
